pkg/allocation/converters/converter_test.go

```diff
@@ -15,27 +15,35 @@
 package converters
 
 import (
+	"fmt"
 	"testing"
 
 	pb "agones.dev/agones/pkg/allocation/go"
 	"agones.dev/agones/pkg/apis"
 	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
 	allocationv1 "agones.dev/agones/pkg/apis/allocation/v1"
+	"agones.dev/agones/pkg/util/runtime"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestConvertAllocationRequestToGameServerAllocation(t *testing.T) {
+	allocated := agonesv1.GameServerStateAllocated
+	ready := agonesv1.GameServerStateReady
+
 	tests := []struct {
 		name               string
 		in                 *pb.AllocationRequest
 		want               *allocationv1.GameServerAllocation
 		skipConvertFromGSA bool
+		features           string
```

Comment:
Since you put this right after name in the test cases, I think you should put it second in the struct as well (so that ordering is consistent and expected).

Reply:
Yeah, looks like I have done that in past. Good catch to be consistent 👍🏻